packages/schematics/angular/library/index.ts

```diff
@@ -156,7 +157,7 @@ function addAppToWorkspaceFile(options: LibraryOptions, workspace: WorkspaceSche
         builder: '@angular-devkit/build-angular:tslint',
         options: {
           tsConfig: [
-            `${projectRoot}/tsconfig.lint.json`,
+            `${projectRoot}/tsconfig.build.json`,
```

Comment:
In our tests we use a `tsconfig.lib.json`, as a counterpart to `tsconfig.app.json`. Can you change this filename please?